otcdocstheme/theme/otcdocs/css.html

```diff
@@ -5,12 +5,6 @@
 <link href="{{pathto('_static/css/fontawesome-all.min.css', 1)}}" rel="stylesheet">
 <link rel="preload" href="{{pathto('_static/webfonts/fa-regular-400.woff2', 1)}}" as="font" type="font/woff2" crossorigin>
 
-<!-- Pygments CSS -->
-<link href="{{pathto('_static/pygments.css', 1)}}" rel="stylesheet">
```

Comment:
Where does this file go? It isn't listed somewhere below

Reply:
it is included automatically by sphinx. Look at any current html - basic and pygments are included in the top (through https://github.com/sphinx-doc/sphinx/blob/4.x/sphinx/themes/basic/layout.html#L94) Now basic is replaced with otcdocstheme